pom.xml

```
@@ -19,7 +19,7 @@
     <version>${revision}${changelist}</version>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/MSTest+Plugin</url>
     <scm>
-        <connection>scm:git:git@github.com:jenkinsci/mstest-plugin.git</connection>
+        <connection>scm:git:https@github.com:jenkinsci/mstest-plugin.git</connection>
```

Need to use a valid URL.
```suggestion
        <connection>scm:git:https://github.com:jenkinsci/mstest-plugin.git</connection>
```

---

Good catch thanks!